Update setup.py

aniketrege opened the pull request:
Added dependency list for setuptools to download automatically, as well as added functionality to automatically fetch required packages in all __init.py__ specified subdirectories.

aniketrege committed:
Update setup.py

Added dependency list for setuptools to download automatically, as well as added functionality to automatically fetch required packages in all __init.py__ specified subdirectories.

mattjegan commented on setup.py:
```diff
@@ -1,16 +1,26 @@
+      'subprocess',
+      'time',
+      'psutil'      
+      ],
```
This should only contain `psutil` like so `install_requires=['psutil'],` since all the other requirements are built into the python standard library and we wouldn't want pip to try to [download clones that may be malicious](https://www.bleepingcomputer.com/news/security/ten-malicious-libraries-found-on-pypi-python-package-index/).

aniketrege replied:
Hi sorry! Will fix that.

mattjegan reviewed (changes requested):
This looks pretty good, thanks for the help. Just a small change needs to be made to get this to work 😄

aniketrege committed:
Update setup.py

Removed redundant packages, and added one for color functionality in HowLong.py

mattjegan commented:
@aniketrege why do we need `termcolor`?

Commit:
Update setup.py
